src/main/kotlin/calculator/main.kt

```diff
@@ -1,12 +1,9 @@
 package calculator
-fun main(args: Array<String>) {
+
+fun main() {
     print("input : ")
-    var input : String? = readLine()
+    var input: String? = readLine()
```

Comment:
```suggestion
    val input: String = readln()
```
[readln](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.io/readln.html)를 활용하면 nullable 타입이 아닌 대신 내부에서 예외를 반환하도록 할 수 있습니다. 또 가변하지 않기 때문에 var로 선언하지 않아도 될 것 같아요!